nikola/conf.py.in

```diff
@@ -343,6 +356,25 @@ HIDDEN_TAGS = ['mathjax']
 # However, more obscure tags can be hidden from the tag index page.
 # TAGLIST_MINIMUM_POSTS = 1
 
+# A list of dictionaries specifying tags which translate to each other.
+# For example:
+#   [
+#     {'en': 'car', 'de': 'Auto'},
+#     {'en': 'window', 'fr': 'fenêtre'},
+#   ]
+# Note that tags in the same language can appear more than once, for
+# example if they have different meaning:
+#   [
+#     {'en': 'cloud', 'de': 'Wolke'},  # the fluffy thing in the sky
+#     {'en': 'cloud', 'de': 'Cloud'},  # cloud computing
+#   ]
+# TAG_TRANSLATIONS = []
+
+# If set to True, a tag in a language will be treated as a translation
+# of the literally same tag in all other languages. Enable this if you
+# do not translate tags, for example.
+# TAG_TRANSLATIONS_ADD_DEFAULTS = False
```

Comment:
It’s a hack: put one value in nikola.py (oh by the way, you have some work to do in this regard) and another in conf.py.in, so new sites get new behavior, and old sites get something safer.